### DISCUSSION TITLE
Profile shows wrong README filename

### DISCUSSION THREAD

**Discussion by toiletbril** [community/community#55272] [Mobile]
| Webpage   | Android app |
| - | - |
| <img src="https://github.com/community/community/assets/36993107/d3ff61dc-5a8b-454c-9f62-7498a258a352" width=400> | <img src="https://github.com/community/community/assets/36993107/bfc8fe0c-2093-4ff4-bee8-bb9d0eb8cc5a" width=400> |

The README repo:
<img src="https://github.com/community/community/assets/36993107/da6b4e02-bfce-494d-9e56-af4f2281b2e7" width=400>

**Comment by jge162** [comment]
you need to use readme.md not txt

**Reply by toiletbril** [reply]
what if i don't like md's font and don't want to put everything into a block of code?

**Reply by jge162** [reply]
Then you will have the issues you are having now. The behavior of the readme will be inconsistent. As the readme with username is a special readme and if you use txt it is basically just a text document. But, ultimately your decision. Bests
